Go/Crypto-Scraper/README.md

```diff
@@ -0,0 +1,21 @@
+# 💲 **CRYPTO SCRAPER**
+A webscraper written in golang, it can scrape prices of the top crypto currencies.
+
+The scraper uses https://coinmarketcap.com/ to get all the data required.
+
+## **👩‍💻Tech Stack**
+- Go
+
+## **📦 Basic Component**
+
+- Go Colly https://pkg.go.dev/github.com/gocolly/colly@v1.2.0
+
+## **💻 Running on Local**
+
+- Fork & Clone the Repo.
+- `cd` to `Go/Crypto-Scraper`.
+- run `go mod Crypto-Scraper`.
+- run `go get github.com/gocolly/colly` in the terminal.
+- After installing the package run `go run main.go` to get the prices of the top crypto currencies.
```

Comment:
@Shivansh-yadav13 represent these column in these manner also add clone project link.
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/156584510-e8322370-6a00-4aad-b518-3368ba5f4367.png">

Reply:
done 👍